.github/workflows/create-tag.yml

```diff
@@ -0,0 +1,20 @@
+# This action creates a release every second Tuesday
+name: "Create and push release tag"
+
+on:
+  workflow_dispatch:
+#  schedule:
+#    "0 10 * * tue/2"
```

Comment:
Did you intend to comment this out?

Reply:
It seems this syntax doesn't work for "every second Tuesday". That's why it's part of the TODO list in the PR description.
(I still pushed it already so both parts of the automation could be reviewed together.)

Reply:
*learns to read*